src/kernel-install/90-loaderentry.install.in

```diff
@@ -39,13 +39,17 @@ else
     ENTRY_DIR="${ENTRY_DIR_ABS#"$BOOT_MNT"}"
 fi
 
+KERNEL_DEST="$ENTRY_DIR_ABS/linux"
+KERNEL_ENTRY="$ENTRY_DIR/linux"
+LOADER_ENTRY="$BOOT_ROOT/loader/entries/$ENTRY_TOKEN-$KERNEL_VERSION.conf"
+
 case "$COMMAND" in
     remove)
         [ "$KERNEL_INSTALL_VERBOSE" -gt 0 ] && \
-            echo "Removing $BOOT_ROOT/loader/entries/$ENTRY_TOKEN-$KERNEL_VERSION*.conf"
+            echo "Removing ${LOADER_ENTRY%.conf}*.conf"
         exec rm -f \
-            "$BOOT_ROOT/loader/entries/$ENTRY_TOKEN-$KERNEL_VERSION.conf" \
-            "$BOOT_ROOT/loader/entries/$ENTRY_TOKEN-$KERNEL_VERSION+"*".conf"
+            "$LOADER_ENTRY" \
+            "${LOADER_ENTRY%.conf}"*".conf"
```

Comment:
Is this bash specific syntax or regular POSIX shell syntax? We need to support regular posix shell for kernel-install.

(We really should just rewrite kernel-install in C but that's a whole different story)

Reply:
`${parameter%word}` is definitely POSIX.